app/test_app/functional/api/test_bulk_disbursement_api.py

```diff
@@ -148,13 +141,16 @@ def test_disbursement_approval_flow(include_list, disbursement_amount, expected_
     """
     # Utility function to create admins to approve
     def make_new_user_and_token(name='francine', tier='superadmin'):
```

Comment:
This part of the test is being janky as hell. I think it just has to do with how we handled sessions in the test special-case. The functionality works fine, it's just this little new user function. Working on a solution.